spartan/devnet-smoke-tests/add_amm_contracts.sh

```diff
@@ -0,0 +1,28 @@
+# Check the amm contract count and create more if current < max_amm_contracts
+amm_contract_count=$(jq '.contracts | map(select(.type=="amm")) | length' state.json)
+max_amm_contracts=1
+amm_contracts_per_run=1
+
+if [ "$amm_contract_count" -lt "$max_amm_contracts" ]; then
+  new_amm_contracts="[]"
+
+  num_new_amm_contracts=$(( amm_contracts_per_run < (max_amm_contracts - amm_contract_count) ? amm_contracts_per_run : (max_amm_contracts - amm_contract_count) ))
+  for i in $(seq 1 $num_new_amm_contracts); do
+    token_0_address=$(aztec-wallet deploy Token --args accounts:test0 Test TST 18 -f test0 --node-url $NODE_URL -a token_0 | grep "Contract deployed at" | awk '{print $4}')
+    token_1_address=$(aztec-wallet deploy Token --args accounts:test0 Test TST 18 -f test0 --node-url $NODE_URL -a token_1 | grep "Contract deployed at" | awk '{print $4}')
+    token_liquidity_address=$(aztec-wallet deploy Token --args accounts:test0 Test TST 18 -f test0 --node-url $NODE_URL -a token_liquidity | grep "Contract deployed at" | awk '{print $4}')
+    amm_address=$(aztec-wallet $([ $i -eq 1 ] && echo "-p native" || echo "") deploy AMM --args "$token_0_address" "$token_1_address" "$token_liquidity_address" -f test0 --node-url $NODE_URL -a amm | grep "Contract deployed at" | awk '{print $4}')
```

Comment:
You could just use the CLI aliases, but nbd. I guess you do need to store these below